stages/org.osbuild.oscap.remediation

```diff
@@ -61,63 +61,64 @@ SCHEMA = """
         "type": "string",
         "description": "The tailoring id"
       },
-      "xml_results": {
-        "type": "string",
-        "description": "Filename for saving the xml result file",
-        "default": "eval_remediate_results.xml"
-      },
       "arf_results": {
         "type": "string",
-        "description": "Filename for saving the arf result file"
+        "description": "Filename for storing the ARF Results file"
       },
-      "html_report": {
+      "xccdf_results": {
         "type": "string",
-        "description": "Filename for saving the html report",
-        "default": "eval_remediate_report.html"
+        "description": "Filename for storing the XCCDF Results file"
       },
       "verbose_log": {
         "type": "string",
         "description": "Filename for verbose error messages"
       },
       "verbose_level": {
         "type": "string",
-        "enum": ["DEVEL", "INFO", "ERROR", "WARNING"],
+        "enum": ["DEVEL", "INFO", "WARNING", "ERROR"],
         "description": "The verbosity level for the log messages"
       }
     }
   }
 }
 """
 
-XML_RESULTS = "eval_remediate_results.xml"
-HTML_REPORT = "eval_remediate_report.html"
-
+DATA_DIR = "/root"
+XCCDF_RESULTS = "oscap_eval_xccdf_results.xml"
+REMEDIATION_SCRIPT = "oscap_remediation.bash"
 
 def main(tree, options):
     # required vars
     config = options["config"]
-    profile = config["profile_id"]
+    profile_id = config["profile_id"]
     datastream = config["datastream"]
     # optional vars
     xccdf_id = config.get("xccdf_id")
-    data_dir = options.get("data_dir")
+    data_dir = options.get("data_dir", DATA_DIR)
```

Comment:
Since we need a place to work with remediation script, the data_dir is now mandatory and defaults to `/root`. Which would be a convenient place to store the remediation artifacts for analyses.